Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "miniscript"
-version = "9.0.2"
-authors = ["Andrew Poelstra <apoelstra@wpsoftware.net>, Sanket Kanjalkar <sanket1729@gmail.com>"]
+version = "9.1.0"
+authors = ["Andrew PoelstrPa <apoelstra@wpsoftware.net>, Sanket Kanjalkar <sanket1729@gmail.com>"]
```

Comment:
In 5b1df514dab84f016d0b7f6758b51c025eef5a9f:

You injected a typo into my name :)